cmd/labs/releases/releases.go

```diff
@@ -0,0 +1,93 @@
+package releases
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"time"
+
+	"github.com/databricks/cli/cmd/labs/localcache"
+	"github.com/databricks/cli/cmd/labs/unpack"
+	"github.com/databricks/cli/libs/log"
+)
+
+func NewReleaseCache(org, repo, cacheDir string) *Releases {
+	return &Releases{
+		cache: localcache.NewLocalCache[Versions](cacheDir, "releases", 1*time.Hour),
```

Comment:
Same as the other one, org and repo should be encoded here. I know the caller has a repo-dedicated cache, but that would cause aliasing between same-named repos from different orgs.